docs/integrations/spark.md

```diff
@@ -389,7 +397,32 @@ On Hadoop 2, configure:
 </configuration>
 ```
 
-(You can also set a default output committer using mapred.output.committer.class and possibly setting mapred.mapper.new-api=false.)
+The lakeFS output committer holds Hadoop working objects on temporary
+branches and merges them back to the output branch when done.  Objects
+appear atomically: it is not possible for another task to see the output
+path with any set of objects except the entire desired output set.
+
+#### Additional options
+
+* For old Hadoop versions you might set a default output committer using
```

Comment:
Older than 2? what are we saying here, that the foc may work for older hadoop versions?

Reply:
I'm trying to say "if you're using mapred rather than mapreduce code".  Changed to talk about "mapred" directly.